client/my-sites/checkout/composite-checkout/components/refund-policies.tsx

```diff
@@ -214,6 +251,13 @@ function RefundPolicyItem( { refundPolicy }: { refundPolicy: RefundPolicy } ) {
 			);
 			break;
 
+		case RefundPolicy.GiftMonthlyPurchase:
+			text = translate(
+				'You understand that gift {{refundsSupportPage}}refunds{{/refundsSupportPage}} are limited to 7 days after purchase or renewal for products with monthly subscriptions.',
+				{ components: { refundsSupportPage } }
+			);
+			break;
+
```

Comment:
Not needed if simplified.

Reply:
You can ignore my comment: p1669821415337469/1669748340.840929-slack-C0117V2PCAE